apps/sports-helsinki/src/pages/index.tsx

```diff
@@ -118,7 +119,9 @@ export async function getStaticProps(context: GetStaticPropsContext) {
       return {
         props: {
           ...(await serverSideTranslationsWithCommon(DEFAULT_LANGUAGE, [
+            'home',
             'search',
+            'event',
```

Comment:
Were translations related to 'event' not working previously?

Reply:
just made them equal like they are everywhere else